rm/st/wlock_helper.c

```diff
@@ -186,8 +186,9 @@ static void _write_lock_put(struct wlock_ctx *wlx)
 M0_INTERNAL void rm_write_lock_put()
 {
 	_write_lock_put(&wlx);
-	wlock_ctx_destroy(&wlx);
 	wlock_ctx_disconnect(&wlx);
+	m0_nanosleep(m0_time(1, 0), NULL);
```

Comment:
Please document this properly as a temporary fix.

Reply:
This is removed. It is not needed.